src/main/java/duke/tasks/ToDo.java

```diff
@@ -0,0 +1,48 @@
+package duke.tasks;
+
+import duke.utils.ResourceHandler;
```

Comment:
Perhaps you could consider putting a space between line 3 and 4 which follow the other classes' import's grouping?